source/data-formats/bson.txt

```diff
@@ -0,0 +1,89 @@
+.. _kotlin-sync-bson:
+
+==========================
+Document Data Format: BSON
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about the BSON data format, how MongoDB
+uses it, and how to install the BSON library independently of the
+{+driver-short+}.
+
+BSON Data Format
+----------------
+
+**BSON**, or Binary JSON, is the data format that MongoDB uses to organize
+and store data. This data format includes all JSON data structure types and
+adds support for types including dates, differently-sized integers, ObjectIds, and
+binary data. For a complete list of supported types, see the
+:manual:`BSON Types </reference/bson-types>` server manual page.
+
+The binary format is not human-readable, but you can use the
+:ref:`BSON library <install-bson-library>` to convert it to the human-readable JSON
+representation. You can read more about the relationship between these
+formats in the :website:`JSON and BSON </json-and-bson>` guide on the MongoDB website.
+
+MongoDB and BSON
+----------------
+
+You can use BSON data with your {+driver-short+} application by using one of the
+following object types that implements the `BSON interface <{+java-api+}/apidocs/bson/org/bson/conversions/Bson.html>`__:
+
+- `Document <{+java-api+}/apidocs/bson/org/bson/Document.html>`__ (BSON library package)
+- `BsonDocument <{+java-api+}/apidocs/bson/org/bson/BsonDocument.html>`__ (BSON library package)
+- `RawBsonDocument <{+java-api+}/apidocs/bson/org/bson/RawBsonDocument.html>`__ (BSON library package)
+- `JsonObject <{+java-api+}/apidocs/bson/org/bson/json/JsonObject.html>`__ (BSON library package)
+
+.. _install-bson-library:
+
+Install the BSON Library
+------------------------
+
+These instructions detail how to add the BSON library as a dependency to
+your project.
+
+.. note::
+
+   If you have already added the {+driver-short+} as a dependency to your
```

Comment:
Q: I'm a little confused on the use case where a user would not have the driver installed but would need to install just the BSON library as a dependency, given that this page talks about using BSON with the user's Kotlin sync application

Reply:
This is a good question – I'm not sure why the other driver versions of this page have this section. I can raise it to the team at large to see if anyone has context.

Reply:
Update: There [have been cases where developers are interested in this](https://www.mongodb.com/community/forums/t/use-of-bson-library-without-driver/115304).

Although developers who find this page after they've already been in the Kotlin docs might not need this, it could be helpful for developers who find the docs through external sources (e.g. googling).